src/main/java/duke/ChatBot.java

```diff
@@ -0,0 +1,170 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+/**
+ * ChatBot is a class that encapsulates what a chat bot has.
+ * It functions as a chat bot that responds to user inputs and
+ * does tasks specified by the user.
+ */
+public class ChatBot {
+
+    private ArrayList<Task> temp;
+    private TaskList tasklist;
```

Comment:
Are these two collections ?

Reply:
Yes they are, just that TaskList is an abstraction of the collection of tasks.